### ISSUE TITLE
Problem apps page in the wiki?

### ISSUE THREAD

**Issue by herrdeh** [microg/android_packages_apps_GmsCore#514]
Many apps are working great with microG - but some do not.

Would it be helpful for the developers to have a "problem apps page" - e.g. in the wiki? - Could be a simple table, where users can share their experience with problems - and possibly solutions.

In the case I find agreement here, I'd be happy to start that wiki page.

**Comment by ArchangeGabriel** [comment]
Seems a good idea. You could list tested version as well.

**Comment by ahstro** [comment]
For the record, I'd love a page like that :smile:

**Comment by herrdeh** [comment]
Right, I tried two samples. Did I pick all info required? - In that case I'd provide a template.

**Comment by deesnook** [comment]
Great idea, thanks a lot!
